gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/impl/PageServiceImpl.java

```diff
@@ -2659,6 +2657,7 @@ private void createOrUpdateChildrenPages(
             }
 
             if (createdOrUpdatedPage == null) {
+                String newPageEntityId = UuidString.generateForEnvironment(environmentId, apiId, pageEntityToImport.getId());
```

Comment:
Do we really have to regenerate the page ID here ?

I believe that, if necessary to regenerate ID, it has been done once in the `preprocessApiDefinitionUpdatingIds``;
And so, here, the id in pageEntityToImport is already the one that has been regenerated.

So maybe it's unecessary to regenerate a `newPageEntityId` again here ; and use pageEntityToImport.getId() instead ?

Reply:
ok